spec/truffle.mspec

```diff
@@ -79,16 +79,19 @@ class MSpecScript
     # Hangs in CI
     "^spec/ruby/library/net",
 
-    # openssl is tested separately as it needs Sulong
+    # openssl and zlib are tested separately as it needs Sulong
```

Comment:
"as it" -> "as they"

Reply:
ah I've modified just half of the sentence